log-server/templates/os.json

```diff
@@ -16,5 +16,10 @@
                 "msg" : { "type" : "text" }
             }
         }
+    },
+    "settings" : {
+        "index" : {
+            "number_of_replicas" : "0"
```

Comment:
Could you leave a comment stating why 0 replicas for now?
Unless you tell me the update for this is coming right-away, only then you can skip the comment :)